lib/datadog/tracing/contrib/redis/instrumentation.rb

```diff
@@ -25,6 +25,20 @@ def call(*args, &block)
                 span.resource = get_command(args, show_command_args)
                 ### BRAZE MODIFICATION
                 span.set_metric Contrib::Redis::Ext::METRIC_RAW_COMMAND_LEN, args.to_s.length
+
+                if Thread.current[Contrib::Redis::Ext::THREAD_GLOBAL_FILEPATH].present?
```

Comment:
`.present?` comes from `active_support`, which is not necessarily active here.

these should probably use `!..nil?`

```suggestion
                if !Thread.current[Contrib::Redis::Ext::THREAD_GLOBAL_FILEPATH].nil?
```